src/_blog/libp2p-IPFSþing-2023-recap.md

```diff
@@ -0,0 +1,137 @@
+---
+tags:
+- libp2p
+title: libp2p at IPFSþing 2023 Recap
+description: Recap of libp2p presentations and workshops at IPFSþing 2023
+date: 2023-05-05
+permalink: "/2023-05-05-libp2p-IPFS%C3%BEing-recap/"
+translationKey: ''
+header_image: /ipfs-thing-2023-logo.svg
+author: Dave Huseby
+---
+
+**Table of Contents**
+
+[[toc]]
+
+## Introduction
+
+Last month, April 15th - 19th 2023, the IPFS community came together in Brussels, Beligium for [IPFSþing 2023](https://blog.ipfs.tech/2023-ipfs-thing-recap/). The libp2p users and contributors community came out to meet up once again to interface with the broader IFPS community as well as share all of the great accomplishements and new work going on in the libp2p project. It was great to reconnect with people from organizations supporting and contributing to libp2p such as [Peergos](https://peergos.org/), [Source Network](https://source.network/), 
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="../assets/ipfs-thing-2023-03.jpg"  width="500">
+        <figcaption style="font-size:x-small;">Group shot of the IPFSþing 2023</a>
+        </figcaption>
+    </figure>
+</div>
+
+Over the course of five days the the libp2p community gave 4 different talks on the recent libp2p developments and finished strong with a workshop where participants built their own peer-to-peer chat application leveraging the same technology as the [Universal Connectivity](https://github.com/libp2p/universal-connectivity) demonstrator project.
+
+## Goals
+
+The goals of the libp2p contributors attending IPFSþing were:
+
+1. Build excitement by demonstrating the [Universal Connectivity application](https://github.com/libp2p/universal-connectivity).
+2. Give updates on the continually improving performace, dealing with non-uniform network topoligy with hole punching, and lowering the barriers to libp2p compatibility.
```

Comment:
```suggestion
2. Give updates on the continually improving performance, dealing with non-uniform network topology with hole punching, and lowering the barriers to libp2p compatibility.
```

Reply:
I don't think we should mention hole punching here. That was something we worked on a long time in the past, let's talk about the new stuff!

Reply:
I pulled out the mention of hole punching.